cli/aws_orbit/remote_files/helm.py

```diff
@@ -260,6 +260,7 @@ def package_team_space_pkg(context: Context, repo: str, team_charts_path: str, t
             "jupyter_team_image": team_context.final_image_address,
             "s3_toolkit_bucket": context.toolkit.s3_bucket,
             "account_id": context.account_id,
+            "utility_data_image": context.images.utility_data.repository,
```

Comment:
repository does not have version.  I gave you the code in slack:` f"{context.images.utility_data.repository}:{context.images.utility_data.version}"`